### PR TITLE
Feature/helm s3

### PR DESCRIPTION
**Pull request by scaneer**
## what
* Adds the helm-s3 plugin to anvil

## why
* Currently using anvil in Jenkins pipeline and need to access helm repository in s3 bucket

## references
* [helm-s3](https://github.com/hypnoglow/helm-s3)

### CODE REVIEW

**Review by flintkr — DISMISSED** [review]

**Comment by bstascavage** [comment]
Automated tests are failing: https://g.codefresh.io/public/accounts/saic/builds/61391f75c9b9db581860222d

**Comment by saic-statonb** [comment]
@scaneer  Seeing this error when I run the tests (locally):

`task: docker run --rm -v $(pwd):/workdir -w /workdir docker.io/saicoss/anvil:local shellspec
Running: /bin/sh [bash 4.4.19(1)-release]
..................F.............F.......

Examples:
  1) printVersion() validates tool is installed by checking version
     call printVersion

     1.1) The status should eq 0

            expected: 0
                 got: 1
            
          # spec/helm_s3_spec.sh:11

  2) printVersion() validates tool is installed by checking version
     call printVersion

     2.1) The status should eq 0

            expected: 0
                 got: 1
            
          # spec/serverless_spec.sh:11

     2.2) [WARNING] It was output to stderr but not found expectation

            stderr: main: line 1478: serverless: command not found
            
          # spec/serverless_spec.sh:8

Finished in 16.32 seconds (user 9.06 seconds, sys 7.36 seconds)
40 examples, 2 failures

Failure examples: (Listed here affect your suite's status)

shellspec spec/helm_s3_spec.sh:8 # 1) printVersion() validates tool is installed by checking version FAILED
shellspec spec/serverless_spec.sh:8 # 2) printVersion() validates tool is installed by checking version FAILED

task: Failed to run task "test": exit status 101`

File: src/docker/Dockerfile [hunk 1]
```diff
@@ -257,6 +257,13 @@ RUN helm plugin install https://github.com/aslafy-z/helm-git.git --version "${HE
   && asdf reshim helm \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+# Install helm-s3 plugin. Get versions from https://github.com/hypnoglow/helm-s3/releases
+ARG HELM_S3_VERSION="=0.10.0"
```
Comment (saic-statonb) [inline src/docker/Dockerfile]: I think the `=0.10.0` should be `0.10.0` in order to pass the spec test.

**Review by saic-statonb — APPROVED** [review]
LGTM - Codefresh pipelines are passing now

File: src/docker/Dockerfile (hunk 1 above)
Comment (scaneer) [reply src/docker/Dockerfile]: oof. Sorry about that

**Pull request merged**